jest.config.js

```diff
@@ -16,9 +16,5 @@ module.exports = {
 		},
 	},
 	roots: ['<rootDir>packages/'],
-	coveragePathIgnorePatterns: [
-		'/node_modules/',
-		'<rootDir>/packages/collection/src/index.ts',
-		'<rootDir>/packages/rest/src/index.ts',
-	],
+	coveragePathIgnorePatterns: ['/node_modules/', '<rootDir>/packages/rest/src/index.ts'],
```

Comment:
```suggestion
	coveragePathIgnorePatterns: ['<rootDir>/packages/rest/src/index.ts'],
```

`node_modules` should always be ignored, even when overriding this array. At least I've personally never  had it scan `node_modules` even when doing so.

Reply:
This actually must stay otherwise it somehow starts traversing `node_modules` lol